### PR TITLE
Update models and admin.py

### PR DESCRIPTION
**Pull request by rosomri**
Hey guys, I made little changes in models-
- Profile model-
-- 'youtube_link' and 'twich_link' - optional fields instead of mandatory.
- UserFollowers model-
--two fields 'follow_from' and 'follow_to'
--these fields have to be **unique together**
--one cannot follow himself
- Stream model-
--'description' field added
--'ends_on' - optional field instead of mandatory
--'ends_on', 'airs_on', 'added_on' - DateTimeField instead of DateField
--ordering - 'airs_on' -> '-added_on' -> 'author'

I also updated the admin.py for making Profile, Stream, and UserFollowers visible.

**I have asked my team members for a review before the Redhaters,** in order for us to make any change if necessary.
Team, please review ASAP, and comment if you think that any change is required.

### CODE REVIEW

File: sellotape/sellotape_dj/main_app/models.py [hunk 1]
```diff
@@ -1,39 +1,49 @@
...
 
 class UserFollowers(models.Model):
-    user_followers_id = models.AutoField(primary_key=True)
-    user_id = models.ForeignKey('self', on_delete=models.CASCADE)
-    follower_id	= models.ForeignKey(Profile, on_delete=models.CASCADE)
+    follow_from = models.ForeignKey(Profile, related_name='follow_from', on_delete=models.CASCADE)
+    follow_to = models.ForeignKey(Profile, related_name='follow_to', on_delete=models.CASCADE)
+
+    class Meta:
+        unique_together = ('follow_from', 'follow_to')
```
Comment (alonweissfeld) [inline sellotape/sellotape_dj/main_app/models.py]: 👌

File: sellotape/sellotape_dj/main_app/models.py [hunk 2]
```diff
@@ -1,39 +1,49 @@
...
-    )   
+    )
+
 
 class UserFollowers(models.Model):
-    user_followers_id = models.AutoField(primary_key=True)
-    user_id = models.ForeignKey('self', on_delete=models.CASCADE)
-    follower_id	= models.ForeignKey(Profile, on_delete=models.CASCADE)
+    follow_from = models.ForeignKey(Profile, related_name='follow_from', on_delete=models.CASCADE)
+    follow_to = models.ForeignKey(Profile, related_name='follow_to', on_delete=models.CASCADE)
```
Comment (alonweissfeld) [inline sellotape/sellotape_dj/main_app/models.py]: I think this is a bit confusing.. how about a simple `user` and `follows` fields?

**Review by alonweissfeld — APPROVED** [review]
Looks good, just see my comment for the UserFollowers naming.

File: sellotape/sellotape_dj/main_app/models.py (hunk 2 above)
Comment (rosomri) [reply sellotape/sellotape_dj/main_app/models.py]: changed to `user_id` and `follows`.

**Review by ItzhakiGuy — APPROVED** [review]

**Pull request merged**